tests/unit/entityInfo.spec.ts

```diff
@@ -53,58 +86,119 @@ describe.skip('Entity Info component with an NR', () => {
     const crumb2 = breadcrumbs.at(2)
     const crumb3 = breadcrumbs.at(4)
 
-    // TODO: add test for "Staff Dashboard" when user is staff
-    expect(crumb1.text()).toStrictEqual('Manage Businesses Dashboard')
+    expect(crumb1.text()).toStrictEqual('Staff Dashboard')
     expect(divider.text()).toStrictEqual('>')
-    expect(crumb2.text()).toStrictEqual('Xyz Ltd.')
-    expect(crumb3.text()).toStrictEqual('BC Benefit Company Incorporation Application')
+    expect(crumb2.text()).toStrictEqual('My Mock Name Inc.')
+    expect(crumb3.text()).toStrictEqual('Correction - Incorporation Application')
   })
-})
 
-describe.skip('Entity Info component without an NR', () => {
-  let wrapper: any
-  let store: any
-
-  beforeEach(() => {
-    store = getVuexStore()
+  it('renders the business name and numbers', async () => {
+    expect(wrapper.vm.$el.querySelector('#entity-legal-name').textContent)
+      .toContain('My Mock Name Inc.')
 
-    // Entity type will always be set with or without an NR
-    store.state.stateModel.tombstone.entityType = 'BEN'
-    // Temp Id will always be set with or without an NR
-    store.state.stateModel.tombstone.businessId = 'T1234567'
-    store.state.stateModel.nameRequest.nrNumber = null
-    store.state.stateModel.nameRequest.details.approvedName = null
+    expect(wrapper.vm.$el.querySelector('#entity-business-number').textContent)
+      .toContain('1234567')
 
-    wrapper = mount(EntityInfo, { vuetify, store })
+    expect(wrapper.vm.$el.querySelector('#entity-incorp-number').textContent)
+      .toContain('BC1234567')
   })
 
-  afterEach(() => {
-    wrapper.destroy()
+  it('renders the business contact information', async () => {
+    expect(wrapper.vm.$el.querySelector('#entity-business-email').textContent)
+      .toContain('mock@email.com')
+
+    expect(wrapper.vm.$el.querySelector('#entity-business-phone').textContent)
+      .toContain('123-456-7890')
   })
+})
 
-  it('renders the Numbered Company header when the EntityType(BC) is present with no NR', async () => {
-    expect(wrapper.vm.$el.querySelector('#nr-header').textContent)
-      .toContain('BC Benefit Company Incorporation Application')
+describe('Entity Info component in a Correction as a numbered company', () => {
+  let wrapper
+  let store
+
+  const mockFiling = {
+    header: {
+      name: 'incorporationApplication',
+      filingId: 12345
+    },
+    business: {
+      identifier: 'BC7654321',
+      legalType: 'BEN'
+    },
+    incorporationApplication: {
+      contactPoint: {
+        email: 'mock@email.com',
+        phone: '123-456-7890'
+      },
+      nameRequest: {},
+      offices: {},
+      parties: [],
+      shareClasses: [],
+      incorporationAgreement: {}
+    }
+  }
 
-    expect(wrapper.vm.$el.querySelector('#entity-legal-name').textContent)
-      .toContain('Numbered Benefit Company')
+  beforeEach(() => {
+    store = getVuexStore()
```

Comment:
Same as above for `store` and state assignments below.

Reply:
Fixed in next commit.